src/client/ws/Gateway.ts

```diff
@@ -34,20 +32,18 @@ export class Gateway {
 				log({ state: 'WS', message: `Sending a heartbeat every ${this.HEARTBEAT_INTERVAL}ms + jitter time` });
 				log({ state: 'WS', message: 'Identifying' });
 
-				this.socket.send(
-					JSON.stringify({
-						op: 2,
-						d: {
-							token: this.token,
-							intents: 33281,
-							properties: {
-								$os: 'linux',
-								$browser: 'Discord iOS',
-								$device: 'ios'
-							}
+				this.send({
+					op: 2,
+					d: {
+						token,
+						intents: this.client.intents,
+						properties: {
+							$os: 'linux',
+							$browser: 'Discord iOS',
+							$device: 'ios'
```

Comment:
```suggestion
							$os: process ? process.platform : 'ailurus',
							$browser: 'ailurus',
							$device: 'ailurus'
```

Reply:
These are discord platforms, would there be any effect on changing them to the library name?

Reply:
On Discord's analytics side of things, they would see ailurus. It has no effect, lots of libraries do this anyways.